### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by Waayway** [sfx101/deck#89]
I have been using deck for a week now and really wished for a dark mode, it seems like deck uses tailwind so it is prob just a config file thing, but i couldn't fine it.

**Comment by sfx101** [comment]
Yeah, I agree with you. Let us see how soon we can release a dark mode update. Ty
